vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -24,6 +24,7 @@ This tutorial covers:
 * A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/) for more information or [install Entando on any Kubernetes provider](../#operations). 
 * Use the Entando CLI command `ent check-env develop` to verify all dependencies.
 * A Jira Service Management account.
+* A [Keycloak server](https://www.keycloak.org/docs/15.0/server_admin/#_email) configured with the your SMTP server settings to enable email verification for users. 
```

Comment:
typo: "the your"

"for users" is unnecessary and confusing if verification is enabled for the user email only; "for the user" is clear (but unnecessary) while "for users" implies a wider audience as much as the general function of the configuration

Reply:
second item, it's the second case